node/src/GlideClusterClient.ts

```diff
@@ -987,6 +996,7 @@ export class GlideClusterClient extends BaseClient {
      */
     public async functionKill(route?: Routes): Promise<"OK"> {
         return this.createWritePromise(createFunctionKill(), {
+            decoder: Decoder.String,
```

Comment:
why we need it hardcoded?

Reply:
This command always returns OK, it is a bit weird option to let user encode OK into bytes

Reply:
I agree with Yury. That's why I just add the decoder without argument.